src/database/src/math.rs

```diff
@@ -0,0 +1,83 @@
+use serde::{Deserialize, Serialize};
+
+pub type NauticalMiles = f64;
+pub type Degrees = f64;
+pub type Radians = f64;
+pub type Feet = f64;
+pub type Metres = f64;

```

Comment:
We usually use american english in code and communication so this should probably be meters? :)

Reply:
Yeah ill change it, in my mind metres is a unit and meters is a group of measurement devices